kaggle/api/kaggle_api_extended.py

```diff
@@ -4643,10 +4643,13 @@ def validate_date(self, date):
     def sanitize_markdown(self, markdown: str) -> str:
         return bleach.clean(markdown)
 
-    def confirmation(self):
-        question = "Are you sure?"
+    def confirmation(self, action: str = ''):
+        if len(action):
+            question = "Are you sure you want to {action}?"
+        else:
+            question = "Are you sure?"
         prompt = "[yes/no]"
-        options = {"yes": True, "no": False}
+        options = {"yes": True, "y": True, "no": False, "n": False}
```

Comment:
optional - This is OK, but I personally like the pattern where `yes` = go ahead and anything else means cancel.  It saves me the trouble of having to type "no" if I messed up the command; I can just mash any keys and it's cancelled.  I might just be exceptionally lazy though 😆